Dockerfile

```diff
@@ -21,7 +21,8 @@ ADD .   /go/src/github.com/coreos/clair/
 WORKDIR /go/src/github.com/coreos/clair/
 
 RUN apk add --no-cache git rpm xz && \
-    go install -v github.com/coreos/clair/cmd/clair && \
+    export CLAIR_VERSION=$(git describe --tag --abbrev=0) && \
```

Comment:
I'm not sure if this is the correct command. When I try this locally, I always get `v2.0.0` returned.

Reply:
I think we both decided that `git describe --dirty` is the command we'll use here.

Reply:
@jzelinskie Yeah this is the old one, I forced pushed to keep everything tidy. Might of made it more annoying then anything...